web/themes/new_weather_theme/assets/js/radar.js

```diff
@@ -1,32 +1,52 @@
-/*
-agenda: {
-  id: 'national', 'weather', 'local' // sets the initial view, default undefined
-  center: [lat, lon] // set the intitial center, default center of Conus
-  location: [lat, lon] // set the initial selected location (only applies to weather view), default undefined
-  layer: {radar product options by view} // the selected product, default to Super Resolution Base Reflectivity
-  filter: 'WSR-88D', 'TDWR', undefined (all) // type of radar stations to display, default to undefined
-  station: {any station id}, undefined // The selected station, default undefined
-}
-opacity: {
-  local/national/local/localStations: 0 to 1 // opacity for each layer, not the if layer is visible (which is set by view)
-}
-menu: true/false // display the top-right menu, default true
-shortFuseOnly: true/false // if only short-fused alerts (vs all hazards) should display (if visible), default false
-animating: true/false // if radar should enable play-back, default false
-basemap: 'standard', 'topographic', 'satallite', 'ocean', 'darkcanvas' // what basemap to display, default 'standard'
-artcc/cwa/county/state/rfc: true/false // if overlay should display, default false
-}
-*/
+const toggleMapExpand = (() => {
+  let expanded = false;
 
-const setupRadar = () => {
+  const sprites = {
+    false:
+      "/themes/new_weather_theme/assets/images/uswds/sprite.svg#zoom_out_map",
+    true: "/themes/new_weather_theme/assets/images/spritesheet.svg#wx_zoom-in-map",
+  };
+
+  return (event) => {
+    const container = event.target.closest(".wx-radar-container");
+    const svgUse = document.querySelector("button.wx-radar-expand svg use");
+
+    expanded = !expanded;
+
+    if (expanded) {
+      container.classList.add("wx-radar-container__expanded");
+    } else {
+      container.classList.remove("wx-radar-container__expanded");
+    }
+
+    const descriptors = Array.from(
+      container.querySelectorAll(".wx-radar-expand__description"),
+    );
+    for (const descriptor of descriptors) {
+      if (descriptor.classList.contains("display-none")) {
+        descriptor.classList.remove("display-none");
+      } else {
+        descriptor.classList.add("display-none");
+      }
+    }
```

Comment:
there is also [`classList.toggle()`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle) which might do the same thing